sound/soc/sof/intel/hda-stream.c

```diff
@@ -185,6 +185,15 @@ hda_dsp_stream_get(struct snd_sof_dev *sdev, int direction)
 			direction == SNDRV_PCM_STREAM_PLAYBACK ?
 			"playback" : "capture");
 
+	/*
+	 * Disable L1 support when there is an active capture stream.
+	 * Workaround to address a known issue with host DMA that results
+	 * in xruns during pause/release.
+	 */
+	if (stream && direction == SNDRV_PCM_STREAM_CAPTURE)
+		snd_sof_dsp_update_bits(sdev, HDA_DSP_HDA_BAR, HDA_VS_EM2,
+					HDA_VS_EM2_L1SEN, 0);
```

Comment:
We might set this several times with the implementation, one for each capture stream_get(), though there is no harm to do that.